components/region_cache_memory_engine/src/engine.rs

```diff
@@ -1,19 +1,31 @@
 // Copyright 2023 TiKV Project Authors. Licensed under Apache-2.0.
 
+use core::slice::SlicePattern;
 use std::{
     collections::BTreeMap,
     fmt::{self, Debug},
     ops::Deref,
     sync::{Arc, Mutex},
 };
 
+use bytes::Bytes;
 use collections::HashMap;
 use engine_traits::{
-    CfNamesExt, DbVector, IterOptions, Iterable, Iterator, Mutable, Peekable, ReadOptions,
+    CfNamesExt, DbVector, Error, IterOptions, Iterable, Iterator, Mutable, Peekable, ReadOptions,
     RegionCacheEngine, Result, Snapshot, SnapshotMiscExt, WriteBatch, WriteBatchExt, WriteOptions,
+    CF_DEFAULT, CF_LOCK, CF_WRITE,
 };
 use skiplist_rs::{ByteWiseComparator, IterRef, Skiplist};
 
+fn cf_to_id(cf: &str) -> usize {
```

Comment:
same with it https://github.com/tikv/tikv/blob/993eb2f610acfc4ee9e6c7b4f03eb63213f6d593/components/raft_log_engine/src/engine.rs#L305

Reply:
Not exactly. CF_RAFT is not included.